edison/README.md

```diff
@@ -15,10 +15,32 @@ make a decision.
 The library is using a Feature Extraction Language (FEX) as a declarative way of defining features. 
 that can be applied to text to generate features. 
 
-## Concepts
-   - Feature extractors
-   - Feature input transformers
-   - Operations
+## Maven dependency
```

Comment:
Do we want to have these details here? Already mentioned in the root readme file